build.gradle

```diff
@@ -207,3 +207,5 @@ apply from: file('gradle/ant-compat/solr.post-jar.gradle')
 
 apply from: file('gradle/node.gradle')
 
+sourceCompatibility = JavaVersion.VERSION_17
+targetCompatibility = JavaVersion.VERSION_17
```

Comment:
Woah... an innocent mistake I presume.
This should be reverted @gerlowskija

Reply:
Yeah, yikes.  Almost certainly this was unintentional.  I'll try reverting this bit and push up a PR assuming there are no issues.

I can't imagine why I would've put this in here 🤔 😬

Reply:
@dsmiley / @gerlowskija should probably make sure to get that revert/change into 9.1 - I'm guessing most people are NOT testing with JDK 11?

I'd be surprised if this change doesn't break running with JDK 11 somehow...

Reply:
It's possibly masked by java.gradle setting sourceCompatibility dynamically to minJavaVersion which is set correctly.

Reply:
ah ok phew - I was like how is this not blowing up on JDK 11 - I swear I had jdk 11 running on main :)